src/tests/ftest/checksum/basic_checksum.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2019 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+
+import ctypes
+import time
+from pydaos.raw import (DaosContainer, IORequest,
+                        DaosObj)
+from apricot import TestWithServers
+from test_utils import TestPool
+
+
+class ChecksumContainerValidation(TestWithServers):
+    """
+    Test Class Description: This test is enables
+    checksum container properties and performs
+    single object inserts and verifies
+    contents. This is a basic sanity
```

Comment:
There is nothing specific to checksum here, so why not using an existing test and add the ability to turn on checksums?

Reply:
Yes, this is a very basic checksum test. All it is trying is enable checksum and insert some data in the objects. I wanted to keep all the checksum testing under one directory. That is the reason for creating this new test case. This is similar to some tests. All those tests are now blocked and needs rework to make it work. I have taken one of those blocked tests and made a subset of test work now.